eox_nelp/processors/xapi/mixins.py

```diff
@@ -0,0 +1,47 @@
+"""Mixin file, this is a collection of classes that could be reused in multiple implementations.
+
+Classes:
+    - BaseCourseObjectMixin: Base mixin for transformers that has as object an edx-platform course
+"""
+from django.utils.functional import cached_property
+from tincan import Activity, ActivityDefinition, LanguageMap
+
+from eox_nelp.processors.xapi import constants as eox_nelp_constants
+from eox_nelp.utils import get_course_from_id
+
+
+class BaseCourseObjectMixin:
```

Comment:
This is a mixin only for transformers?

Reply:
that is the intention

Reply:
I said because I thought the name would same something with transformers